### PR TITLE
Fix authentication bugs

### LINKED ISSUE #225: fix authorization issues

### PR DESCRIPTION
**Pull request by Vucis**
closes #225

### CODE REVIEW

File: frontend/src/pages/project/FetchProjects.tsx [hunk 1]
```diff
@@ -1,114 +1,125 @@
...
-    }
-
-    ));
-    formattedData = formattedData.flat()
-    return formattedData
+      })
+    );
+    formattedData = formattedData.flat();
+    return formattedData;
   } catch (e) {
```
Comment (Gerwoud) [inline frontend/src/pages/project/FetchProjects.tsx:122]: error e is now thrown so can be removed i think?

**Review by Gerwoud — COMMENTED** [review]
what is this? :skull:

File: frontend/src/pages/project/FetchProjects.tsx (hunk 1 above)
Comment (Vucis) [reply frontend/src/pages/project/FetchProjects.tsx]: [e82d4fe](https://github.com/SELab-2/UGent-3/pull/226/commits/e82d4fef2cdf445ed820019ac6bc2f7382fb21d3)

**Review by Gerwoud — APPROVED** [review]
looks :fire: to me

**Pull request merged**